mmv1/third_party/terraform/services/bigquery/resource_bigquery_table.go.tmpl

```diff
@@ -1834,7 +1839,14 @@ func resourceBigQueryTableRead(d *schema.ResourceData, meta interface{}) error {
 	datasetID := d.Get("dataset_id").(string)
 	tableID := d.Get("table_id").(string)
 
-	res, err := config.NewBigQueryClient(userAgent).Tables.Get(project, datasetID, tableID).Do()
+	tableMetadataView := d.Get("table_metadata_view").(string)
```

Comment:
How is `if tableMetadataView, ok := d.GetOk("table_metadata_view"); ok { ... }`?